src/components/molecules/Chatbox/components/ChatboxThreadControls/ChatboxThreadControls.tsx

```diff
@@ -7,16 +7,18 @@ import "./ChatboxThreadControls.scss";
 export interface ChatboxThreadControlsProps {
   closeThread: () => void;
   threadAuthor?: string;
+  text?: string;
 }
 
 export const ChatboxThreadControls: React.FC<ChatboxThreadControlsProps> = ({
   closeThread,
   threadAuthor,
+  text = "",
```

Comment:
can we leave it `undefined` as we do for `threadAuthor` ?

Reply:
That would be preferable.